src/main/java/service/UserService.java

```diff
@@ -0,0 +1,32 @@
+package service;
+
+import db.Database;
+import model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class UserService {
+    private static final Logger logger = LoggerFactory.getLogger(UserService.class);
+
+    /**
+     * User 생성
+     *
+     * @param user
+     */
+    public void create(User user) {
+        Database.addUser(user);
+        logger.debug("\n UserId: {} \n, UserPassword:{}\n, UserName:{}\n, UserEmail:{}\n", user.getUserId(), user.getPassword(), user.getName(), user.getEmail());
```

Comment:
```suggestion
        logger.debug("Create user successed!\n UserId: {} \n, UserPassword:{}\n, UserName:{}\n, UserEmail:{}\n", user.getUserId(), user.getPassword(), user.getName(), user.getEmail());
```

이렇게 수정해보자는 의미였습니다. 아랫줄은 지우고요.

Reply:
https://github.com/kakao-2022-summer/be-w234-was/pull/59/commits/c5355e3aabdacc50e38475fd427d3b62ebe9ae79 여기서 수정이된거죠?